docs/user/CLI.md

```diff
@@ -23,18 +23,18 @@ Option                                Description
 -e, --environment <File>              initial global environment (optional)
 -h, --help                            prints this help
 -i, --input <File>                    input file, requires the query option (optional)
+-if, --input-format <InputFormat>     input format, requires the query option -- [PARTIQL (default), ION]
```

Comment:
Is the default PARTIQL or ION? The default in [main.kt](https://github.com/partiql/partiql-lang-kotlin/blob/f33050e6a7f09ad7b9318af19f9ea1c5084bae4d/cli/src/org/partiql/cli/main.kt#L105) is ION. If the defaults are PARTIQL, may need to update the above text (and the [--help text](https://github.com/partiql/partiql-lang-kotlin/blob/f33050e6a7f09ad7b9318af19f9ea1c5084bae4d/cli/src/org/partiql/cli/main.kt#L50-L68)) to include the --input-format option.